### DISCUSSION TITLE
Pressing Ctrl-H while viewing a replay will hide the replay settings sidebar with no visual feedback.

### DISCUSSION THREAD

**Discussion by WitherFlower** [ppy/osu#31357] [Ideas]
That got me in a state of wondering where the hell it went, only to find out that I had accidentally pressed the hotkey after asking in the discord server.

Thinking about it, now that options have their own sidebar, that hotkey is quite useless, so maybe having the sidebar openable at all times is fine.

**Comment by peppy** [comment]
Not pressing ctrl-h is another option.

This hotkey is a legacy one only intended for screen recording.
